spec.html

```diff
@@ -35420,7 +35672,18 @@ <h1>Notation</h1>
         <p>The descriptions below use the following internal data structures:</p>
         <ul>
           <li>
-            A <dfn id="pattern-charset" variants="CharSets">CharSet</dfn> is a mathematical set of characters. In the context of a Unicode pattern, “all characters” means the CharSet containing all code point values; otherwise “all characters” means the CharSet containing all code unit values.
+            A <dfn>CharSetElement</dfn> is one of the two following entities:
+            <ul>
+              <li>
+                If _rer_.[[UnicodeSets]] is *false*, then a CharSetElement is a character in the sense of the Pattern Semantics above.
+              </li>
+              <li>
+                If _rer_.[[UnicodeSets]] is *true*, then a CharSetElement is a sequence whose elements are characters in the sense of the Pattern Semantics above. This includes the empty sequence, sequences of one character, and sequences of more than one character. A sequence of length 1 is the same as a single character.
```

Comment:
```suggestion
                If _rer_.[[UnicodeSets]] is *true*, then a CharSetElement is a sequence whose elements are characters in the sense of the Pattern Semantics above. This includes the empty sequence, sequences of one character, and sequences of more than one character.
```

I don't think this sentence adds anything.

Reply:
Addressed as part of https://github.com/mathiasbynens/ecma262/pull/21. PTAL